app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -0,0 +1,151 @@
+angular.module('topologyApp', ['kubernetesUI','ui.bootstrap'])
+.config(['$httpProvider', function($httpProvider) {
```

Comment:
This bit should be replaced by `miqHttpInject` so that we don't copy the same code all over the place.. (Assuming you meant to create a new module, and not extend `topologyApp`, else you can remove it.)